app/konnect/analytics/use-cases.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Report Use Cases
+content_type: tutorial
+badge: plus
+---
+
+Custom reporting in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to CSV. 
+Let's go through some example situations where you could leverage custom reports.
+
+In this scenario, you just joined an organization as an API product manager. 
+Your first task is to create a few reports that model business KPIs so that the executive team and investors have a grasp on the state of the organization's APIs. 
+You've selected the following KPIs because they would give a brief, but wholistic, understanding of the APIs: 
+
+* **Number of requests:** This metric measures the total number of requests made to the API. 
+This is good for understanding total usage of an API, and can potentially be used to uncover scalability requirements. 
+* **Average throughput:** Throughput is the measurement of the number of requests an API can handle per second. 
```

Comment:
I am not sure about using "average" in this concent and also "per second". We do have a "throughput" metric but that's not average it's just counts the number of request **per minute**. Maybe remove the "average" and change "seconds" to "minutes".

That's probably more for what you may want to look into https://www.moesif.com/blog/technical/api-metrics/API-Metrics-That-Every-Platform-Team-Should-be-Tracking/#4-request-per-minute-rpm.